src/trackers/SN.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, config):
         self.tracker = 'SN'
         self.source_flag = 'Swarmazon'
         self.upload_url = 'https://swarmazon.club/en/upload/upload.php'
-        self.forum_link = 'https://swarmazon.club/php/forum.php?forum_page=2-swarmazon-rules'
```

Comment:
This URL needs to stay as forum authentication is integrated to the sites authentication system. Directing forum page request to this URL makes sure the user is still logged to the forum before redirecting.